### PR TITLE
NR-62327: added obfuscation expression

### PR DESCRIPTION
**Pull request by NSSPKrishna**
This PR contains
* Added Log Configurations pkg
* Generated code for Obfuscation Expression
* Added Obfuscation Expression test

### CODE REVIEW

File: pkg/logconfigurations/types.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	//
+	Name string `json:"name,omitempty"`
+}
+
+type obfuscationExpressionsResponse struct {
+	Actor Actor `json:"actor"`
+}
+
+// DateTime - The `DateTime` scalar represents a date and time. The `DateTime` appears as an ISO8601 formatted string.
+type DateTime string
```
Comment (mbazhlekova) [inline pkg/logconfigurations/types.go:92]: We usually override the `DateTime` scalar with `nrtime.DateTime`. Take a look at this example in the `tutone.yml` https://github.com/newrelic/newrelic-client-go/blob/main/.tutone.yml#L366

**Comment by mbazhlekova** [comment]
Looks good! Just one change and we can merge.

File: pkg/logconfigurations/types.go (hunk 1 above)
Comment (NSSPKrishna) [reply pkg/logconfigurations/types.go]: done

**Review by mbazhlekova — CHANGES_REQUESTED** [review]

File: pkg/logconfigurations/obfuscation_expression_test.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/newrelic/newrelic-client-go/v2/pkg/config"
+)
+
+func Example_basic() {
```
[the remark this answers is not in the document]
Comment (NSSPKrishna) [reply pkg/logconfigurations/obfuscation_expression_test.go]: done

**Review by mbazhlekova — APPROVED** [review]

**Pull request merged**